.github/CONTRIBUTING.md

```diff
@@ -16,11 +16,16 @@ cd vercel
 corepack enable
 pnpm install
 pnpm build
-pnpm lint
-pnpm test-unit
 ```
 
-Make sure all the tests pass before making changes.
+To run the tests, we'll need a `VERCEL_TOKEN`. To get that token:
+
+- Option 1: Login to https://vercel.com, go to Settings, Tokens, then create a "Full Account" token to use for this purpose.
+- Option 2: If you've logged in with the `vercel` CLI before, you can find the token value in `~/Library/Application Support/com.vercel.cli/auth.json`.
+
+Save that token or export it in your terminal profile.
+
+Then run the tests with `pnpm turbo run test`.
```

Comment:
This will only run a couple tests, I dont think this is what you want

https://github.com/vercel/vercel/blob/873327e9bdb56be5dbfcf3ca45308d9f8308debc/package.json#L42

Reply:
Also you need to run the `gen.js` script to set the proper cache key

Reply:
I wasn't aware of `gen.js`. I've never run it. Is it something that needs to be manually run?

Reply:
Every script runs it before turbo https://github.com/vercel/vercel/blob/873327e9bdb56be5dbfcf3ca45308d9f8308debc/package.json#L43

Reply:
See https://github.com/vercel/vercel/pull/8830